javaparser-testing/src/test/java/com/github/javaparser/manual/BulkParseTest.java

```diff
@@ -69,8 +73,29 @@ public void bulkTest(SourceRoot sourceRoot, String testResultsFileName) throws I
             }
             return DONT_SAVE;
         });
+        writeResults(results, testResultsFileName);
+    }
+
+    public void bulkTest(SourceZip sourceRoot, String testResultsFileName) throws IOException {
+        sourceRoot.setJavaParser(new JavaParser(new ParserConfiguration().setValidator(new Java9Validator())));
```

Comment:
@ryan-beckett I kind of regret not having that base class now - I had to write the same thing twice for directories and zip files...